README.md

```diff
@@ -1,109 +1,12 @@
-# Eppo IOS Feature Flagging SDK
+# Eppo iOS Feature Flagging SDK
 
 iOS implementation of the Eppo Randomization and Feature Flagging SDK.
 
-## 1. Install the SDK
+## Integrating the SDK
 
-While in XCode:
+https://docs.geteppo.com/feature-flags/sdks/client-sdks/ios/
```

Comment:
Suggest making this a Markdown link

```
Please refer to our [official documentation](tps://docs.geteppo.com/feature-flags/sdks/client-sdks/ios/)
```

etc

Reply:
Good idea!